source/secure_tunneling.c

```diff
@@ -165,14 +165,17 @@ static void s_init_websocket_send_frame_options(
     AWS_ZERO_STRUCT(frame_options->rsv);
 }
 
-static int s_secure_tunneling_send_data(struct aws_secure_tunnel *secure_tunnel, const struct aws_byte_buf *data) {
+static int s_secure_tunneling_send(
+    struct aws_secure_tunnel *secure_tunnel,
+    const struct aws_byte_buf *data,
```

Comment:
Consider data can be NULL when type is `STREAM_START` and `STREAM_RESET` and code accordingly.